CONTRIBUTING.md

```diff
@@ -1,70 +1,98 @@
+* 🤝 **Calm, Human Reviews** — *Assume good intent* and give **specific**, **curious**, and **modest** feedback. Reviews are for learning together, not for scoring points.
+* 🛡️ **Open Stewardship & Clear Trail** — Be transparent while respecting stewardship: welcome participation, let roles form through practice, and keep history readable (we use [***Conventional Commits***](https://www.conventionalcommits.org/en/v1.0.0/) to help with that).
+
+### ✅ How We Review
 
-- **Minimal by design** — Space is held for emergent agreements.
-- **Human-first, Agile-friendly** — We value presence over process.
-- **Conventional Commits** — Please use [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) format for commit messages (`feat:`, `fix:`, `docs:`, etc.)
-- **Open by invitation** — We value transparency, but also stewardship.
+* **Responds with clarity and curiosity**
+* **Offers specific suggestions rather than judgments**
+* **Remains modest and open to being changed by another’s point of view**
+* **Treats reviews as a shared opportunity for practice and growth**
+
+### 🤝 Joining the Community
+
+If you’re new here, you’re welcome to **reach out to an existing member** when starting your first contribution. There is *no formal mentorship structure* — just a space for connection and shared learning. Let the conversation grow naturally, and see what emerges.
 
 ---
 
 ## 🧭 Roles in the Dojo
 
-Roles are voluntary and reviewed periodically to ensure mutual clarity and care. Cultivating a role is an opportunity for deeper training and responsibility—not a requirement for contribution or community engagement.
+Roles are *voluntary* and reviewed periodically to ensure mutual clarity and care. Cultivating a role is an opportunity for deeper training and responsibility — **not** a prerequisite for contribution or community engagement.
+
+### ✅ Trusted Committer
 
-### ✅ Trusted Committer (Purple Belt+)
-- Demonstrates technical fluency and principled contribution
-- Understands the codebase and the ethos of the program
-- May merge PRs and curate discussions
-- Operates as a steward and ambassador
+* **Qualification:** ***Purple Belt*** in the program
+* **Demonstrates** technical fluency and principled contribution
+* **Understands** the codebase and the ethos of the program
+* **May** merge PRs and curate discussions
+* **Operates** as a steward and ambassador
 
-### ⚡ Project Maintainer (Black Belt+)
-- Curates both the **codebase** and the **program structure**
-- Embodies the intention of the Dojo at a higher level of intention
-- Guides others in principled contribution and pattern expression
-- Observes and evolves the connective tissue between project and program
+### ⚡ Project Maintainer
+
+* **Qualification:** ***Black Belt*** in the program plus a deep mindset exercise
+* **Curates** both the **codebase** and the **program structure**
+* **Embodies** the intention of the Dojo at a higher level of practice
```

Comment:
intention of Mindset Dojo

you wouldn't say the Facebook

could also replace Dojo with the program and community (but Mindset Dojo, perhaps bold and or italicize)